src/components/30-organisms/modal/index.js

```diff
@@ -0,0 +1,91 @@
+import { LitElement, html, css, unsafeCSS, svg } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+import closeIcon from '@axa-ch/materials/icons/close.svg';
+
+/* eslint-disable import/no-extraneous-dependencies */
+
+//    If you need other axa-XXX components inside your new component,
+//    import them here like this:
+//
+//    import myDependentComponent1 from '@axa-ch/XXX;
+
+import {
+  defineVersioned,
+  /* versionedHtml, */
```

Comment:
Not sure what that is. Please check if other components import this and otherwise remove this line.

Reply:
commercial-hero-banner uses this import versionedHtml and use it in the render() method as `return versionedHtml(this)`